### DISCUSSION TITLE
translations/{id} Api brailleStatus 질문

### DISCUSSION THREAD

**Discussion by jun-brro** [sunnybraille/sunnybraille-server#98] [General]
현재 client에서는 progress를 1000ms(1초) 단위로 확인하고 있습니다.

<img width="279" alt="image" src="https://github.com/sunnybraille/sunnybraille-server/assets/115399447/aa0836d4-09ce-43df-8b3b-1b1f15ba1bc7">

OCR Progress는 100으로 진행되고 있으나, brailleTranslationPercentDone 단은 업데이트가 되지 않는 현상이 있어, 이 부분을 확인 요청드립니다

@gitchannn

**Comment by gitchannn** [comment]
죄송함다 코드상에서 제가 수정을 하지 않았기 때문임당 금방 수정해서 다시 알려드릴게요 ><

**Reply by gitchannn** [reply]
코드 수정했습니다!

https://github.com/sunnybraille/sunnybraille-server/pull/100

<img width="688" alt="image" src="https://github.com/sunnybraille/sunnybraille-server/assets/107979804/53420ca4-b73a-422b-aba5-54ecf5ad6a38">
